tests/console/commands/test_update.py

```diff
@@ -0,0 +1,57 @@
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+import pytest
+
+from poetry.packages import Locker
+from tests.helpers import get_package
+
+
+if TYPE_CHECKING:
+    from poetry.poetry import Poetry
+    from tests.helpers import TestRepository
+    from tests.types import CommandTesterFactory
+    from tests.types import FixtureDirGetter
+    from tests.types import ProjectFactory
+
+
+@pytest.fixture
+def source_dir(tmp_path: Path) -> Path:
+    return Path(tmp_path.as_posix())
+
+
+@pytest.fixture
+def poetry_with_old_lockfile(
+    project_factory: "ProjectFactory", fixture_dir: "FixtureDirGetter"
+) -> "Poetry":
+    source = fixture_dir("old_lock")
+    pyproject_content = (source / "pyproject.toml").read_text(encoding="utf-8")
+    poetry_lock_content = (source / "poetry.lock").read_text(encoding="utf-8")
+    return project_factory(
+        name="foobar",
+        pyproject_content=pyproject_content,
+        poetry_lock_content=poetry_lock_content,
+    )
+
+
+def test_dry_run_update(
```

Comment:
Can you please add similar tests (or extend existing tests) for the following commands (if not yet existing):

- `add --dry-run`
- `add --lock --dry-run`
- `remove --dry-run`

Further, it seems that tests for `add` check that the `pyproject.toml` is not changed for dry-run. Can you please add this check here, too?

Reply:
And probably, you should rebase first before adding tests. 🙏